index.html

```diff
@@ -253,5 +253,81 @@ <h3 class="aboutheading">About me</h3>
       </ul>  
     </section>
 
+<!-- Contact form -->
```

Comment:
[OPTIONAL] - For future references, kindly consider Using imperative mood in the subject line of your commit messages. Example –` Add fix for dark mode toggle state` 🤓 . Here is an awesome [article](https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/) to help you with this 📖